typescript/github-proxy/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@hyperlane-xyz/github-proxy",
+  "description": "Github proxy that adds the API key to requests",
+  "version": "5.1.0",
+  "scripts": {
+    "deploy": "wrangler deploy",
+    "dev": "wrangler dev",
+    "start": "wrangler dev",
+    "test": "vitest",
+    "prettier": "prettier --write ./src ./test",
+    "cf-typegen": "wrangler types"
+  },
+  "type": "module",
+  "homepage": "https://www.hyperlane.xyz",
+  "repository": "https://github.com/hyperlane-xyz/hyperlane-monorepo",
+  "keywords": [
+    "Hyperlane",
+    "Github",
+    "Proxy",
+    "Typescript"
+  ],
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@cloudflare/vitest-pool-workers": "^0.4.5",
+    "@cloudflare/workers-types": "^4.20240821.1",
+    "@faker-js/faker": "^8.4.1",
+    "chai": "4.5.0",
+    "prettier": "^2.8.8",
+    "typescript": "^5.5.2",
```

Comment:
Can we use the same version we're already setting in the SDK + Utils? In general, it's good for tooling across the monorepo to be as consistent as possible. If you'd like to upgrade everything to 5.5.x, that's fine with me :)

Reply:
Sure upped everything to 5.5.2. Also updated in registry